### PR TITLE
Makman2/linterpack6

### PR DESCRIPTION
**Pull request by Makman2**

### CODE REVIEW

File: bears/xml2/XMLBear.py [hunk 1]
```diff
@@ -20,41 +20,49 @@ def path_or_url(xml_dtd):
...
-    diff_message = "XML can be formatted better."
-    output_regex = r'(.*\.xml):(?P<line>\d+): (?P<message>.*)\n.*\n.*'
-    gives_corrected = True
-    use_stderr = True
+@linter(executable='xmllint',
+        use_stdout=True,
+        use_stderr=True)
+class XMLBear:
+    """
+    Checks the code with ``xmllint``.
```
Comment (Asnelchristian) [inline bears/xml2/XMLBear.py]: maybe add a link

File: bears/xml2/XMLBear.py [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+from coalib.bearlib.abstractions.Linter import linter
 from coalib.bears.requirements.DistributionRequirement import (
     DistributionRequirement)
 from coalib.settings.Setting import path, url
 
 
 def path_or_url(xml_dtd):
     '''
-    Coverts the setting value to url or path.
+    Converts the setting value to url or path.
```
Comment (Asnelchristian) [inline bears/xml2/XMLBear.py]: i think this should be in a different commit

**Comment by AbdealiJK** [comment]
XMLBear is failing in CI, can you cherry pick the others on to https://github.com/coala-analyzer/coala-bears/pull/723 so they can get merged ?

**Comment by Asnelchristian** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/733/commits/183a4aaae35435abd2fea651e6404da218a03adb

**Comment by Makman2** [comment]
@AbdealiJK already did that: https://github.com/coala-analyzer/coala-bears/pull/723

and yeah I don't get this damn bear working... behaves totally weird

File: bears/xml2/XMLBear.py (hunk 2 above)
Comment (Makman2) [inline bears/xml2/XMLBear.py]: done :)

File: bears/xml2/XMLBear.py (hunk 1 above)
Comment (Makman2) [inline bears/xml2/XMLBear.py]: added

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/733/commits/2c510d3a2c50fe9909b25fddb78ee47e1b27bc5a

**Comment by sils1297** [comment]
unack https://github.com/coala-analyzer/coala-bears/pull/733/commits/bd52cb79a8c272785eacf74d451b980c108bc3b6 commit message doesn't start upper case after colon

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/733/commits/8aa2eb0f991206e0d31d79254a3e5910088efcbe

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/733/commits/5e0a88828df275753ccd7c1252fb2042a12ba412

File: bears/xml2/XMLBear.py [hunk 3]
```diff
@@ -20,41 +20,50 @@ def path_or_url(xml_dtd):
...
-            xml_schema: path="",
-            xml_dtd: path_or_url=""):
-        '''
-        Checks the code with ``xmllint``.
+    _output_regex = re.compile(r'.*:(?P<line>\d+): (?P<message>.*)\n.*\n.*')
 
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         xml_schema: path = "",
+                         xml_dtd: path_or_url = ""):
```
Comment (sils1297) [inline bears/xml2/XMLBear.py]: no space around = please

**Comment by sils1297** [comment]
unack https://github.com/coala-analyzer/coala-bears/pull/733/commits/f0960a8edf8a0b171bb3d4acf44c5ce346d692b2

File: bears/xml2/XMLBear.py (hunk 3 above)
Comment (Makman2) [inline bears/xml2/XMLBear.py]: ouhhh yes

**Comment by Makman2** [comment]
reack
45ec128 XMLBear: Add link to xmllint site
cc41da6 XMLBear: Use double-quotes for docstring
ee835db XMLBearTest: Add comment

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/733/commits/8cbd7ed93405b97cbe5c2f7c309bc3f18b29e43a

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/733/commits/955d55293e8a2a96b2427af8d2c5de4cba6b1e79

**Comment by sils1297** [comment]
@rultor merge
